submissions/team-turing.md

```diff
@@ -0,0 +1,49 @@
+
+### **What is the instance space?**
+We can think of the instance space as the supermarket products that are then transposed as the possible compositions of images, specifically the color value of each pixel in the images that the camera at the self-checkout captures, but more generally we can think of it as the shape and color on the image, perhaps as well as the measured weight of the item.
+
+### **What are the concepts c(x)?**
+The concepts could then be the products sold at the supermarket for example specific vegetables or fruit that certain images form, for example images of a butternut squash (a composition of pixels in the shape and color of butternut squash) with certain weights (for example as here weighing 1.215 kg) that form the concept butternut squash. For products with barcodes the concept is primarily labeled based on the barcode. When you scan the barcode, the system knows the weight from their database and confirms if it is the right concept on the second scale. 
+
+### **What hypothesis h(x) can we use?**
+The hypotheses that we can use are the compositions of the images that the self-checkout camera captures and weight and whether they are considered to estimate the target concept, an apple-shaped, apple-red object weighing 0.170 kg represents an apple, or at least is deemed most likely to represent a red apple or an red onion etc.
+
+### **What is the target distribution D?**
+The target distribution would be the probability of a certain fruit or vegetable to be measured at the self-checkout which would not be uniformly distributed as they are not all equally likely to be measured at the self-checkout but rather dependent on the demand and sales of the fruits and vegetables.
+
+### **What is the oracle EX[c,D]?**
+The oracle could be a procedure within the self-checkout system that can be called while the learning algorithm is learning a concept c and returns a labeled example of an instance, image that has been labeled as specific fruit or a vegetable, drawn randomly according to sales numbers.
+
+### **How do we measure the error?**
+We could measure the error according to the selection made by the customer after measuring the item, whether the customer selects the label chosen for the measured item. We assume that the options given on the screen are given with certain probabilities, the 6-12 labeled deemed most likely to correspond to the item measured in descending order from the left. Therefore the label given with the highest probability for the ginger is incorrectly classified as potatoes in bulk or in the case of the scallion where no positively labeled option is offered so the customer must manually select the correct label, which shows that the given hypotheses disagree with the target concept.
```

Comment:
This is then saying correct answer is only in first place, i.e. it gives the correct answer the highest probability? or would it be more wrong if the customer needs to search for the product?

Reply:
Both are valid ways to estimate the error. However, for this use case I think having a user need to use the keyboard for searching for their item is a worse UX, there for I'd rather consider something like a stepwise error estimate: 

0 if correctly in first place
1 if item is in the 2-6th place
2 if item is in 7-12th place
10 if item is not in the list (so heavily weighing false negative classifications)

These numbers are adhoc, and would need to be adjusted to the needs of Krónan.

Reply:
My main reasoning for not emphasizing the correct classification in the first place of the list, is that in some cases it's going to be really hard for the classifier to correctly estimate the right one, for instance "red apple" and "janagold apple" are very similar considering the limited features (weight form the scale, simple image with fast image processing). 

So classifying the object correctly as "apple" is probably good-enough for the business case of Krónan, not necessarily which type of apple.